lib/Backends/CPU/Transforms.cpp

```diff
@@ -54,6 +54,11 @@ static Node *optimizeCPUConv(ConvolutionNode *CN, Function *F) {
     return nullptr;
   }
 
+  // This optimization is not supported with Dilation currently
```

Comment:
period

Reply:
Are all the comments supposed to end with period?  It was not apparent from many other comments.

Reply:
Yeah comments should be properly punctuated https://llvm.org/docs/CodingStandards.html#commenting

Reply:
Thanks for the link, that makes it clear.